### ISSUE TITLE
404 Not found.

### ISSUE THREAD

**Issue by mzch** [dicedtomatoreal/zipline#47]
Well, I uploaded `Myself.png` and got http://uixns.com/u/MySelf.png but 404 not found.

_Originally posted by @mzch in https://github.com/ZiplineProject/zipline/issues/46#issuecomment-717756306_

**Comment by dicedtomatoreal** [comment]
Hello I was not able to reproduce this, so im not sure what the problem is.

**Comment by dicedtomatoreal** [comment]
Doublecheck your config for any issues that might have caused this.

**Comment by mzch** [comment]
I re-installed zipline freshly, so Syntax Error occurred:

<img width="1491" alt="ScreenShot" src="https://user-images.githubusercontent.com/621178/97767964-fd93d100-1b62-11eb-8a1a-0bbd4190bf73.png">

**Comment by mzch** [comment]
- Zipline.toml
```toml
[database]
type = "postgres"
host = "127.0.0.1"
port = 5432
database = "zipline"
username = "zipline"
password = "password"

[meta]
title = "Zipline"
description = "mzch Zipline Server"
thumbnail = "https://github.githubassets.com/images/modules/open_graph/github-mark.png"
color = "#128377"

[core]
secret = "secret"
port = 8000

[uploader]
directory = "/srv/zipline/files"
route = "/u"
length = 6
original = true
blacklisted = []

[urls]
route = "/s"
length = 4
vanity = true
```

- Node.js: v14.15.0
- npm: 6.14.8

**Comment by mzch** [comment]
I shortened `https://github.com/awesome-selfhosted/awesome-selfhosted` and got `https://uixns.com/go/NO5q`, but 404 not found, too.
Why zipline generates `/go` instead of `/s`?

**Comment by dicedtomatoreal** [comment]
its been changed to /go as the default, but if u set it in your config it should work?

**Comment by dicedtomatoreal** [comment]
I dont really know what the problem is since i cant reproduce it, sorry if you're able to find out make a PR!

**Comment by mzch** [comment]
Yes, in my `Zipline.toml` I put `urls.route` with `/s` as above. When I visist `https://uixns.com/s/NO5q`, it works. Only urls on web UI are incorrect.

**Comment by mzch** [comment]
Fixed #48
